stock_valued_picking_report/report/stock_picking_valued_report.xml

```diff
@@ -13,7 +13,8 @@
                 <th class="text-right"><strong>Taxes</strong></th>
             </t>
         </xpath>
-        <xpath expr="//span[@t-field='pack_operation.product_uom_id']/.." position="after">
+        <xpath expr="//tr[@t-foreach='o.pack_operation_ids']"
```

Comment:
Ok, although the previous alternatives access a more specific position:

` <xpath expr="//span[@t-field='pack_operation.qty_done_uom_ordered']/.."`
or 
`<xpath expr="//span[@t-field='pack_operation.product_uom_id'][last()]/.."`

Reply:
Very interesting ...thanks!

Reply:
Please change it to a more specific one as indicated, or the order can change.

Reply:
Changes done

Reply:
But the position is still "inside". Can this be after or before an element for not depending on the inheritance for the order?

Reply:
Yes, last commit better referenced with an "after"